pkg/k8shandler/kibana/route.go

```diff
@@ -7,11 +7,18 @@ import (
 	"github.com/openshift/elasticsearch-operator/pkg/utils"
 	"github.com/sirupsen/logrus"
 	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/client-go/util/retry"
 
+	consolev1 "github.com/openshift/api/console/v1"
 	route "github.com/openshift/api/route/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	AppLogsConsoleLinkName   = "kibanaAppPublicURL"
+	InfraLogsConsoleLinkName = "kibanaInfraPublicURL"
```

Comment:
these need to be all lowercase
https://github.com/openshift/elasticsearch-operator/pull/257#issuecomment-608068523